systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlApiST.java

```diff
@@ -121,15 +122,30 @@ void testUserTasks() {
         assertThat(response, containsString(CruiseControlUserTaskStatus.COMPLETED.toString()));
     }
 
+    @Order(5)
+    @Test
+    void testMetrics() {
```

Comment:
Just a question -> shouldn't be this test in `MetricsST`? I know you already added test to this suite, but I'm curious. It can be really confusing for the first time.

Reply:
I agree, as `/metrics` is not actually a CC API endpoint this test would be better in the MetricsST.

Reply:
I would say that name of the suite `CruiseControlApiST.java ` will tell explicitly the user that tests are designed to tests the APIs which CC provide or it is bind.

Reply:
Yeah, I don't think that testing metrics is ok here. This class is about testing CC API endpoints which `/metrics` isn't.